ckanext/datitrentinoit/model/mapping.py

```diff
@@ -100,13 +100,12 @@ def create_base_dict(guid, metadata, config):
 
     def dateformat(d):
         return d.strftime(r"%Y-%m-%d")
-        return d.isoformat()
 
-    start_date = metadata.get_anno_inizio() or '1970'
-    if len(start_date) < 4:
-        log.warn(f"Bad annoinizio found: '{start_date}'")
-        start_date = '1970'
-    created = datetime.datetime(int(start_date), 1, 1)
+    start_date = metadata.get_anno_inizio()
+    end_date = metadata.get_anno_fine()
+
+    created = datetime.date(int(start_date), 1, 1) if start_date else '1970'
```

Comment:
won't this line create different formatted string according to the existance of start_date?

Reply:
Hi @etj 
sorry Im not getting your point ?

Reply:
The formatting changes according to the value of start_date, e.g.:
- start_date = None --> created = "1970"
- start_date = 2000 --> created = "2000-01-01"

This is not even the original behaviour for a field that should not be involved in this issue.